package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "spectacle",
-  "version": "4.0.4",
+  "version": "4.0.5",
```

Comment:
Please don’t bump the version, since we’ll bump it as we push out new releases :)